tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -5121,9 +5121,11 @@ class HLSLExternalSource : public ExternalSemaSource {
       const TemplateArgument &arg = argLoc.getArgument();
```

Comment:
Is the comment above still accurate that this only checks that types are primitive types? I'm not sure it was true when it was committed. :/

Reply:
I will remove the inaccurate part.